src/main/groovy/life/qbic/datamodel/datasets/OxfordNanoporeMeasurement.groovy

```diff
@@ -78,11 +79,13 @@ final class OxfordNanoporeMeasurement {
     }
 
     private void assessPooledStatus() {
```

Comment:
If nested if/nullsafe cases must be used, please add informative comments

Reply:
i did not touch this line of code in this PR, nore was it part of this fix PR.

Reply:
sorry, meant the following lines, see new comment